app/assets/images/payment_icons/latitude_creditline_nz.svg

```diff
@@ -0,0 +1 @@
+<svg height="24" viewBox="0 0 38 24" width="38" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" role="img" aria-labelledby="pi-latitude_creditline_nz"><title id="pi-latitude_creditline_nz">Latitude CreditLine (NZ)</title><clipPath id="pi-latitude_creditline_nz_a"><path clip-rule="evenodd" d="m35 0h-32a2.946 2.946 0 0 0 -3 3v18a3.009 3.009 0 0 0 3 3h32a2.946 2.946 0 0 0 3-3v-18a3.009 3.009 0 0 0 -3-3z"/></clipPath><rect fill="#9c4532" height="24" rx="3" width="38" x="-88.925" y="-30.13"/><g clip-path="url(#pi-latitude_creditline_nz_a)"><image height="638" transform="matrix(.038 0 0 .038 0 .034)" width="1013" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAHIAAABICAMAAAAd4lGEAAAAXVBMVEVia3FhanFha3FianFhanAAi81eZ25gaW9cZWxlbnNjbHJia3L///+Xl5qkpafPz9Dt7u7a2tvl5eW0tLb5+fnCw8T09PSEio57godtdXsAhMdze4AAhslocHYAickjGMEjAAAE8UlEQVR4XuyUSYrdQBBEfYCIHIf6Pdj3P6ZJ1OCVG4TdYPBPAqn0guRpVd/mS+b19cdv5x9XPpVP5VNZv46fUNRfUwKbaz6jYajBh7rwB0ooKVNTtaA2S01IxwWWoBjZsLKMGnRe3ZZ3ldC3jnMKMAxgm/UfRn9XYGrZYIoe5oQdx5R7oebaqHvKqqMAtIJstDD7JAp5lrpSpimGPMJwl5c0rlLVRE4jhI17SsQxhFax56gw3iQeCkhiBjjHkpPpj9DHSDtjlQMRe3E9tmuN+0olk5lHsyEKNiCyyj2SKclFIRqCS5npAkhT8uQ95eAkAPZxU5NLqUA/AKRLgjIeIgAj1Vc5W2UQI0o19bqpjMf+KESawQQbp1GQk3wYE8qWtofki+077TDFU/xgTqQ0HfeUAxM2qpIK9dLYTCEpswDKHDhTRx0tpiLUcNMaNeza/asAwEztc1Ob/b7A9su2/sg1tdW19t9d608lvmTe33+2Y645joMwAI6Nx0BygBAh7/2PuXaMgG5GU0b7t67Kyylf/Eic5gfkEGx90CGEp8I08xrAWLMehhK/vhaQDwLMhACTZua+Sj/TN0hoCISORO9m028+whM5rCMdLiG7MagjAOhuGgJkWnjx7IsrAMLd2hfeITtxmAjf+RrRbXQsAjYiuEO8aeeMb5Cq70wcg0EfAILud+jmWws4ztxN/REJHQkjdMHRnroeWWwro+uHQvuGofof5Af5QX6QhgkqznBsF196CBwzFhvagUvI7wsDDWhTYLef8k7f1pTQZqvI+UfghKmwQNfjRpLpXncSjmNXkTARsc+grxlnXqQoOdLR5q0F94ff1sNKvRw+xKlIt8JPbUru1cyXRPdtL600xx3fVxJUgZdQPisZ+BACkewiZqfp3LlDFi+SwSJWoX9mYEZoFyPaIhFKrZmOXsp7Pe1JtJ4+zPm6hDmmeAPFZokOVXBi0zNnkeM6i6jkY3LHupVTChBLPVWK5EuIkhSfsSpKlVpKzdlaER3F/YWB7i4EgLB8XVK6FKCizVlTn+lYThfTmEgW2aMh/OlnCKw8+/S0sY1rjnE3U6sR6x5TLmeJfhZZNd6WayfZCDcMZhtCy60W6vdWOjKS2cZEbLSL71niqsR8FiklpqSH5JRUdW1x2/2H6B1gR8KqY/nSzfmG255yFo6so3Kkel5RlTFupWSmqGqi+ULqaQ8bIm6wYCWp2MbJLVYzr8uMNI/uZqnAWaghIykyUr8k0FGeQsu3dUOyIZVP4EiduY1xL+e2vyKFp3swNC75U30AWHSsWlmTDUJH3jYeOispP5HdzKncLD6tt4NZdFe+Ry2W1Yg7mx+TPJG2N4aRQb+vl5YVlVOMKVXP2LPo9M5ZMiS8IHsxg8e9B94hARuTRSFClJXhl3/dt00ULcwPK6EHD1vlGrISSzDZ0m6W+fdKSdpMx9CQRzkbknEyKYQZuP5UgBtmuQxzXWpMBM7VZjUzBc6lEsWtqoKoFuFtFC6wj8PhV7EEC9rGQJyL5y6nXWRn3z1Gb+EOO81unDpcRaJvqqizilWT4gCMzD5QjFJG65geRfR/7LCMBI/lwaQ0kxp5+Mj3md9dvEpDwhxJXEofJRycpKpI4gO3h8zONDX2AOIURER8+1Qwbw6cvPw//7e3DsPjodXJPvIeF9Pn897n897n897ne/kLEWG//FzSFNAAAAAASUVORK5CYII="/></g><path d="m0 3v18a3.009 3.009 0 0 0 3 3h32a2.946 2.946 0 0 0 3-3v-18a3.009 3.009 0 0 0 -3-3h-32a2.946 2.946 0 0 0 -3 3zm35-2a2.006 2.006 0 0 1 2 2v18a2.006 2.006 0 0 1 -2 2h-32a2.006 2.006 0 0 1 -2-2v-18a2.006 2.006 0 0 1 2-2z" fill-rule="evenodd" opacity=".1"/></svg>
```

Comment:
This logo looks pixelated. Can we make it vectorized?
[![alt](https://screenshot.click/15-03-x5fds-pw6r8.png)](https://screenshot.click/15-03-x5fds-pw6r8.png)